OWNERS_ALIASES

```diff
@@ -25,13 +26,31 @@ aliases:
     - tomassedovic
     - iamemilio
     - pierreprinetti
+  openstack-reviewers:
   vsphere-approvers:
     - abhinavdahiya
     - dav1x
+    - mtnbikenc
+    - jcpowermac
     - staebler
+  vsphere-reviewers:
+  baremetal-approvers:
+    - hardys
+    - stbenjam
   baremetal-reviewers:
     - celebdor
     - markmc
     - russellb
     - hardys
     - stbenjam
+  aws-approvers:
+    - patrickdillon
```

Comment:
If we drop auto approval we'll need to add a second here so PRs from @patrickdillon aren't blocked.

Reply:
well i'm thinking we should use the <platform>-approvers as additive to installer-approvers to reduce duplication...